private/rules/kt_jvm_export.bzl

```diff
@@ -88,8 +85,8 @@ def kt_jvm_export(
         name = name,
         maven_coordinates = maven_coordinates,
         lib_name = lib_name,
-        deploy_env = updated_deploy_env,
```

Comment:
deploy_env here gets added back into the docs so we were generating docs for kotlin stdlib which seems wrong. Changed to excluded_workspace to keep it out of the docs. This is also what we do with scala already.